app/views/welcome/about.html.erb

```diff
@@ -0,0 +1 @@
+<div class='container'>
    <div class= 'about-nav'>
      <u>About</u>
      <%= link_to "Home", "/" %>
    </div>
    <div class='about'>
      <div class='about-body'>
        <h2>About She’s Coding</h2>
          <p>
            She’s Coding is an open-source website project currently under development in cooperation with the
            documentary film CODE: Debugging the Gender Gap. She’s Coding is meant to serve as the destination
            for anyone who wants to learn more about the gender gap problem in computer science and become part
            of the solution. Whether you’re a woman currently working in tech, a girl interested in learning to code,
            a male ally for women in computer science, or a company trying to find ways to increase diversity, you’ve
             come to the right place!
          </p>
      <h2>Wait, open-source?</h2>
        <p>
          Yes! We are setting up an open-source Ruby on Rails project on Github, and de- fining some
           simple rules on how to contribute. She’s Coding will be built by the community,
           for the community. The initial site will launch in mid April 2015 to coincide
           with the world premiere of the CODE documentary, and will continue to grow and
           expand in resources and functionality over time. Want to learn more or be a
           part of the she’s coding team? Contact us!
        </p>
        <p>
          Convert understanding into action – Join the She’s Coding team!
        </p>
    </div>
  </div>
</div>

```

Comment:
Below, you have a link to 'Home', next to the 'About' in the 'about-nav'. This should not be a link to home, but a link to 'How'. So under the same route you will get to 'About' and 'How'. 'How' will have content like the links to the tools we use for collaborating on the project (Github, Trello, Google Group, etc.), for now you can just add any dummy content.

So the idea is that one sees the 'About' section when clicking on 'About' in the main navbar. Then one sees the 'How' content when clicking on the 'How' link in the 'about-nav', and 'About' in that 'about-nav' turns into a link.